[LiveComponent][doc] Document how to deal with locales (#239)
symfony/ux: src/LiveComponent/src/Resources/doc/index.rst

    # config/routes.yaml
    live_component:
        resource: '@LiveComponentBundle/Resources/config/routing/live_component.xml'

>         prefix: '/{_locale}' # comment out if your app is not localised

Reviewer: ```suggestion
        # uncomment to add localization to your components
        #prefix: '/{_locale}'
```

I'd prefer to comment by default.
